docs/client/concepts/parameter-stores.mdx

```diff
@@ -32,5 +32,5 @@ If you've had this problem already, you'll recognize the power immediately.  If
 Parameter Stores are available in:
 - Android SDK v4.33.0+
 - iOS SDK v1.45.0+
-- @statsig/js-client and @statisg/react-bindings v1.4.0+
+- @statsig/js-client and @statsig/react-bindings v1.4.0+
```

Comment:
@tore-statsig is this a typo or intended?

Reply:
nice fix, thanks